doc/ref_impl/cntt-ri2/chapters/chapter01.md

```diff
@@ -29,19 +29,41 @@ In order to promote the quick deployment of the RI and avoid duplicating effort
 <a name="1.1.2"></a>
 ### 1.1.2 Structure of the document
 
-> To be completed
+The document first consolidates the requirements that need to be addressed by this RI in [Chapter 2](./chapter02.md). [Chapter 3](./chapter03.md) then details the requirements that need to be met in order for anyone to be able to install the RI, whether that be into a community lab, their own corporate or personal lab or some other environment such as CNF Testbed or a public cloud provider. An operational runbook is documented in [Chapter 4](./chapter04.md) to aid the deployment of the RI into a chosen environment and includes step-by-step instructions for a selection of installers as well as steps on how to validate the deployment. Finally, [Chapter 5](./chapter05.md) is a placeholder to allow the documentation of any gaps found during the development of this document and the associated artefacts.
 
 <a name="1.1.3"></a>
 ### 1.1.3 Terminology
 
+Terminology in this document will follow [CNTT Terminology](../../../tech/glossary.md).
+
 <a name="1.2"></a>
 ## 1.2 Scope
 
+The scope of this document is as follows:
+1. To generate eco-system requirements for the establishment of the RI, including labs, tooling, installers, releases and automation requirements
+1. Provide detailed description file definition for installers, VNF features, and etc..
```

Comment:
can this be changed to CNF?

Reply:
Done in https://github.com/cntt-n/CNTT/pull/1651/commits/b4545b35f0a35b8c6e71dee39c087e6c58fe5db6